src/app/sidebar-right/components-right/report/report.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MapService } from 'src/app/shared/services/map.service';
+
+@Component({
+  selector: 'app-report',
+  templateUrl: './report.component.html',
+  styleUrls: ['./report.component.scss']
+})
+export class ReportComponent implements OnInit {
+  currentClick!: object;
+
+  constructor(private _mapService: MapService) { }
+
+  ngOnInit(): void {
+    this._mapService.clickPoint.subscribe((point: object) => {
+			this.currentClick = point; 
+		});
+  }
```

Comment:
This is fine for now - just a note that in the future, we'll probably subscribe to updates to the workflows and catch the workflow outputs to display in the report depending on the value type